operator-sdk-bin/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: Stephan Bechter <daolis@apogeum.at
+pkgname=operator-sdk-bin
+pkgver=0.18.1
+pkgrel=1
+epoch=
+pkgdesc="SDK for building Kubernetes applications. Provides high level APIs, useful abstractions, and project scaffolding."
+arch=("x86_64")
+url=""
+license=('Apache 2.0')
+groups=()
+depends=()
+makedepends=()
+checkdepends=()
+optdepends=()
+provides=()
+conflicts=()
+replaces=()
+backup=()
+options=()
+install=
+changelog=
```

Comment:
i am a huge fan on removing empty lines, they are not necessary and clutter the file